configs/SBMs_node_clustering_GatedGCN_PATTERN_PE_500k.json

```diff
@@ -33,6 +33,6 @@
         "dropout": 0.0,
         "batch_norm": true,
         "pos_enc": true,
-        "pos_enc_dim": 40
+        "pos_enc_dim": 2
```

Comment:
The `pos_enc_dim` here should be `20`, or `2`?

Reply:
It is 2 for PATTERN (not 40). Thanks

Reply:
Thanks for confirming. Ready to merge.